store/queryAuctions.js

```diff
@@ -126,13 +117,12 @@ function queryAuctionId(from, to, showAuctions = false, itemId, callback) {
   }
 
   if (Number.isNaN(Number(from)) || Number.isNaN(Number(to))) {
-    return callback({ error: "parameters 'from' and 'to' must be integers" });
+    throw new TypeError("Parameters 'from' and 'to' must be integers");
   }
-  redis.zrangebyscore(itemId, from, to, (error, auctions) => {
-    if (error) {
-      logger.error(error);
-    }
-    const object = {
+
+  try {
```

Comment:
I think we should add another layer of caching here, since many concurrent requests seem to cause a memory overflow.

Reply:
You can add another cachedFunction call here.